File: week6/mario.py
from cs50 import get_int

while True:
    height = get_int("Height:")
    if height > 0 and height < 9:
        break

for i in range(1, height + 1):
    # Print (height - i) blanks 
    for j in range(height - i):
        print(" ", end="")
    # Print i hashes
    for j in range(i):
        print("#", end="")
    # Print two blanks in the middle
    print("  ", end="")
    # Print i more hashes
    for j in range(i):
        print("#", end="")
    # Print newline
    print()

File: week1/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{

    int height;
    
    // Ensure that the given number is beteen 1 and 8 inclusive
    do 
    {
        height = get_int("Enter height: ");
    } 
    while (height < 1 || height > 8);

    // Loop across each row
    for (int i = 1; i <= height; i++)
    {
    
        // Print (height - i) blanks 
        for (int j = 0; j < height - i; j++)
        {
            printf(" ");
        }
       
        // Print i hashes
        for (int j = 0; j < i; j++)
        {
            printf("#");
        }
        
        // Print two blanks in the middle
        printf("  ");

        // Print i more hashes
        for (int j = 0; j < i; j++)
        {
            printf("#");
        }
        
        // Print newline
        printf("\n");
    }
}
